include/usb/class/usb_msc.h

```diff
@@ -46,8 +46,7 @@
 #ifndef __USB_MSC_H__
 #define __USB_MSC_H__
 
-/** MSC Class, Subclass and Protocol Codes */
-#define MASS_STORAGE_CLASS		0x08
```

Comment:
Why? All definitions for MSC should be here inside usb_msc.h.

Reply:
I think there is a difference between "Definition related to Mass Storage Class" and class differentiator like MASS_STORAGE_CLASS, COMMUNICATION_DEVICE_CLASS, etc. If we want to handle some general code and we need this class of device, assuming we have tens of classes, we might include tens of headers?

Reply:
You're right in that case it would not be stupid, but when it would be the case? I know only one place where you need all class codes, in usb_descriptor. But there is WIP to change it so that the respective descriptor is declared in the function and the usb_descriptor does not have to be changed with each new function.

Reply:
There is also logically nicer looking code when you combine logical definitions together, I bet this is done the same way in the Linux kernel. But I can also remove duplicated definition from common code, where do we move other classes then?

Reply:
I have reworked usb_dfu and webusb and they are lying around somewhere in github. But I can imagine that your suggestion is not so bad, but then usb_common must be touched every time when a new class will be added. Now I am undecided...